knowledge_base/save_job_result_to_volume/src/query.ipynb

```diff
@@ -0,0 +1,60 @@
+{
```

Comment:
This should render fine in the Github repo files. For example: https://github.com/databricks/cli/blob/main/internal/testdata/notebooks/py1.ipynb